From 9bc03040261277c93ffe2461b7e729be290e506a Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:06:56 -0400
Subject: [usb] uas: Add Hans de Goede as uas maintainer

Message-id: <1410887463-20674-345-git-send-email-dzickus@redhat.com>
Patchwork-id: 95361
O-Subject: [RHEL7 PATCH 344/591] uas: Add Hans de Goede as uas maintainer
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit f50a4968deb7bf38c46f5baf62db9431a099531a
Author: Hans de Goede <hdegoede@redhat.com>
Date:   Mon Oct 28 10:48:04 2013 +0000

    uas: Add Hans de Goede as uas maintainer

    At the kernel-summit Sarah Sharp asked me if I was willing to become the
    uas maintainer. I said yes, and here is a patch to make this official.

    Also remove Matthew Wilcox and Sarah Sharp as maintainers at their request.

    I've also added myself to the module's author tag, so that if people look there
    rather then in maintainers they will know they should bug me about uas too.

    Signed-off-by: Hans de Goede <hdegoede@redhat.com>
    Signed-off-by: Sarah Sharp <sarah.a.sharp@linux.intel.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/MAINTAINERS b/MAINTAINERS
index c7ae9fa..e7d3fcd 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -8534,8 +8534,7 @@ S: Maintained
 F: drivers/net/wireless/ath/ar5523/
 
 USB ATTACHED SCSI
-M: Matthew Wilcox <willy@linux.intel.com>
-M: Sarah Sharp <sarah.a.sharp@linux.intel.com>
+M: Hans de Goede <hdegoede@redhat.com>
 M: Gerd Hoffmann <kraxel@redhat.com>
 L: linux-usb@vger.kernel.org
 L: linux-scsi@vger.kernel.org
diff --git a/drivers/usb/storage/uas.c b/drivers/usb/storage/uas.c
index ad97615..08e9710 100644
--- a/drivers/usb/storage/uas.c
+++ b/drivers/usb/storage/uas.c
@@ -2,6 +2,7 @@
  * USB Attached SCSI
  * Note that this is not the same as the USB Mass Storage driver
  *
+ * Copyright Hans de Goede <hdegoede@redhat.com> for Red Hat, Inc. 2013
  * Copyright Matthew Wilcox for Intel Corp, 2010
  * Copyright Sarah Sharp for Intel Corp, 2010
  *
@@ -1261,4 +1262,5 @@ static struct usb_driver uas_driver = {
 module_usb_driver(uas_driver);
 
 MODULE_LICENSE("GPL");
-MODULE_AUTHOR("Matthew Wilcox and Sarah Sharp");
+MODULE_AUTHOR(
+ "Hans de Goede <hdegoede@redhat.com>, Matthew Wilcox and Sarah Sharp");
-- 
1.7.1